Gruntfile.js

```diff
@@ -60,10 +60,10 @@ module.exports = function(grunt) {
         ],
         tasks: [
           'jshint',
-          'clean:dev',
-          'compass:dev',
+          'clean:dist',
+          'compass:dist',
           'handlebars',
-          'concat:dev'
+          'concat:dist'
```

Comment:
great catch.  i added in a call to `build_img` (which calls both `clean:img` then `smushit`)